frontend/src/app/modules/project/components/create-project/create-project.component.ts

```diff
@@ -130,12 +131,19 @@ export class CreateProjectComponent implements OnInit, OnDestroy {
     }
 
     public getErrorMessage(field: string): string {
-      const control = this.projectForm.get(field);
-      const isMaxError: boolean = !!control.errors && !!control.errors.max;
-      return control.hasError('required')
-        ? 'Value is required!'
-        : (isMaxError)
-          ? `Quantity must be less than ${control.errors.max.max}!`
-          : 'validation error';
+        const control = this.projectForm.get(field);
+        const isMaxError: boolean = !!control.errors && !!control.errors.max;
+        const isMinLengthError: boolean = !!control.errors && !!control.errors.minlength;
+        const isMaxLengthError: boolean = !!control.errors && !!control.errors.maxlength;
+
+        return control.hasError('required')
+                ? 'Value is required!'
```

Comment:
Please, rewrite this to switch/case or if/else block

Reply:
I thought about "switch", but did not find the corresponding enum. I'll try again.

Reply:
I did it with "If/else". Check it!)